src/components/SettingsLoader.vue

```diff
@@ -1,18 +1,52 @@
 <template>
     <div id="settings-loader">
-        <slot v-if="isLoaded" />
+        <slot v-if="phase === PHASES.LOADED" />
 
-        <div v-else :class="['modal', 'z-top', {'is-active': error}]">
+        <div v-if="phase > PHASES.LOADED" class="modal z-top is-active">
             <div class="modal-content">
                 <div class="notification is-danger">
                     <h3 class="title">Error</h3>
                     <h5 class="title is-5">{{error && error.name}}</h5>
                     <p>{{error && error.message}}</p>
-                    <div v-if="error && error.solution">
-                        <br/>
-                        <h5 class="title is-5">Suggestion</h5>
-                        <p>{{error && error.solution}}</p>
+                    <br />
+                    <h5 class="title is-5">Suggestion</h5>
+
+                    <p v-if="phase === PHASES.GAME_FAILED">
+                        This is a problem with the mod manager itself.
+                        If there's a newer version of the manager
+                        available, try installing it.
+                    </p>
+
+                    <div v-else-if="phase === PHASES.SETTINGS_FAILED">
+                        <p>
+                            Loading of local user settings failed. You
+                            can use the button below to reset the
+                            settings, but note that all settings for all
+                            games will be lost and this can't be undone.
+                            This won't affect the loaded mods or mod profiles.
+                        </p>
+                        <br />
+                        <button @click="resetSettings" class="button is-white">
+                            Reset settings
+                        </button>
                     </div>
+
+                    <p v-else-if="phase === PHASES.RESET_FAILED">
+                        Well, that's awkward. Resetting of the settings
+                        failed as well. You can still try to reset the
+                        settings manually by following these
+                        <a target="_blank" rel="noopener noreferrer"
+                            href="https://github.com/ebkr/r2modmanPlus/wiki/Error:-White-or-blank-game-select-screen-on-startup#corrupted-settings-on-update">
+                            instructions.
+                        </a>
+                    </p>
+
+                    <p v-else-if="phase === PHASES.RETRY_FAILED">
+                        That didn't do much, it seems. Locally stored
```

Comment:
See previous comment about wording, I think we want to avoid "friendly" wording and to keep it more professional and straight to the point